### DISCUSSION TITLE
MySQL and Mojo full app

### DISCUSSION THREAD

**Discussion by ziyati** [mojolicious/mojo#1712] [Q&A] [answered]
Hi,
I have some problems to connect a full App (mojo generate app Hello) to MySQL using DBI ,Mojo::Moysql

need help please.

**Comment by kraih** [comment, marked as answer]
For us to be able to help you need to actually say what the problem is.

**Reply by ziyati** [reply]
Thanks for interest, 
when i create a project mojo, what should i do to connect this app with a table in mysql and then display data in template. 
need steps or tutorial.
Friendly

**Reply by kraih** [reply]
Use the included [example apps](https://github.com/jhthorsen/mojo-mysql/tree/master/examples) then.

**Reply by ziyati** [reply]
Thank you so much :)
